spec/generate_field_yaml/generate_field_yaml_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'marc_cleanup'
+require 'byebug'
+
+RSpec.describe 'generate_field_yaml' do
+  describe 'sample MARC rule' do
+    let(:source_fname) { "#{Dir.getwd}/spec/fixtures/generate_field_yaml/marcrules.txt" }
```

Comment:
After checking out this PR, I needed change this path to: "#{Dir.getwd}/../../spec/fixtures/generate_field_yaml/marcrules.txt" (up two levels)